options/elf/include/elf.h

```diff
@@ -144,6 +176,17 @@ __MLIBC_INLINE_DEFINITION unsigned char ELF64_ST_INFO(unsigned char bind, unsign
 	return (bind << 4) | type;
 }
 
+typedef struct {
+  Elf64_Half si_boundto;
+  Elf64_Half si_flags;
+} Elf64_Syminfo;
+
+#define ELF32_ST_BIND(val)		(((unsigned char) (val)) >> 4)
```

Comment:
These should probably be made inline to match `ELF64_ST_*`.

Reply:
No longer in this PR, moved to #547